cookbooks/bcpc_kafka/attributes/default.rb

```diff
@@ -52,9 +49,9 @@
   broker[:dual][:commit][:enabled] = false
   broker[:offsets][:storage] = 'kafka'
 
-  # Default to a 0.10.0 protocol for 0.11.x upgrades.
-  broker[:inter][:broker][:protocol][:version] = '0.10.0'
-  broker[:log][:message][:format][:version] = '0.10.0'
+  # Default to a 1.1.0 protocol.
+  broker[:inter][:broker][:protocol][:version] = '1.1.0'
+  broker[:log][:message][:format][:version] = '1.1.0'
```

Comment:
Should the 1.1.0 be another attribute like our active release notion in the Hadoop code?